TUTORIAL.md

```diff
@@ -257,7 +257,16 @@ See  Help in the **Snippet Generator** for the `withEnv` step for more details o
 
 ## Build Parameters
 
-If you have configured your pipeline to accept parameters when it is built — **Build with Parameters** — they are accessible as Groovy variables of the same name.
+If you have configured your pipeline to accept parameters when it is built — **Build with Parameters** — they are accessible as Groovy variables inside `params`. Example:
+
+```groovy
+node {
+  sh "isFoo is ${params.isFoo}"
+  sh "isFoo is " + params.isFoo
+  if (params.isFoo) {
```

Comment:
Might want to note that this assumes the `isFoo` parameter was defined a boolean parameter (checkbox in the UI). Well, maybe Groovy will treat nonempty string parameters as truish, not sure offhand.